src/components/Heading/Heading.tsx

```diff
@@ -46,6 +47,14 @@ type Props = {
   variant?: Variant;
 } & React.HTMLAttributes<HTMLHeadingElement>;
 
+const getComputedSize = (size: HeadingSize) => {
```

Comment:
oh now that I'm reading the code again, this should probably be named `getComputedAs` 🤔 my b